website/layouts/_default/list.html

```diff
@@ -24,7 +24,7 @@ <h1 class="c-page-header__title e-heading e-heading__1">{{.Title}}</h1>
 <article class="{{ with .Content }}c-copy{{end}}">
   {{ .Content }}
   <ol class="c-timeline">
-    {{ range .Pages.Reverse }}
+    {{ range .Pages }}
```

Comment:
This leads to 1-0-0 being reverse-chronological, but 1-1-0 being chronological? That feels buggy?

Reply:
Hmm, no it's because the real dates are in the future so Hugo won't let me publish them. I mean, I could rewrite the page but I think once the real dates are in they will both be the same.

Reply:
Can we give them fake non-equal dates? e.g. January 1, January 2, ...?

Reply:
For what purpose? Can you explain more?

Reply:
Oh, wait, now I understand what you're saying. Yes I'll fix this